### PR TITLE
Add gcp_sql_database_instance_details closes #38

### LINKED ISSUE #38: Add gcp_sql_database_instance_details
**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

### PR DESCRIPTION
**Pull request by ParthaI**
### Checklist
- [ ] Issue(s) linked

### CODE REVIEW

**Review by rajlearner17 — APPROVED** [review]
Approved to merge in staging

File: dashboards/database/sql_database_instance_detail.sp [hunk 1]
```diff
@@ -0,0 +1,556 @@
...
+  container {
+
+    graph {
+      title     = "Relationships"
+      type      = "graph"
+      direction = "TD"
+
+      nodes = [
+        node.gcp_sql_database_instance_node,
+        node.gcp_sql_database_instance_to_machine_type_node,
```
Comment (rajlearner17) [inline dashboards/database/sql_database_instance_detail.sp:80]: gcp_sql_database_instance_to_machine_type_node can be removed

File: dashboards/database/sql_database_instance_detail.sp [hunk 2]
```diff
@@ -0,0 +1,556 @@
...
+        node.gcp_sql_database_instance_to_data_disk_node,
+        node.gcp_sql_database_instance_to_kms_key_node
+      ]
+
+      edges = [
+        edge.gcp_sql_database_instance_to_machine_type_edge,
```
Comment (rajlearner17) [inline dashboards/database/sql_database_instance_detail.sp:86]: edge.gcp_sql_database_instance_to_machine_type_edge be removed

File: dashboards/database/sql_database_instance_detail.sp [hunk 3]
```diff
@@ -0,0 +1,556 @@
...
+
+  sql = <<-EOQ
+    select
+      i.kms_key_name as id,
+      k.name as title,
+      jsonb_build_object(
+        'KMS Key Name', i.kms_key_name,
+        'KMS Key Version Name', i.kms_key_version_name,
+        'Primary', k.primary,
+        'Version Template', version_template
```
Comment (rajlearner17) [inline dashboards/database/sql_database_instance_detail.sp:523]: Just keep this as we have GCP > KMS > Key (both node & edge)

**Review by karanpopat — COMMENTED** [review]
LGTM

**Pull request merged**